doc/languages-frameworks/agda.section.md

```diff
@@ -27,9 +30,65 @@ agda.withPackages (p: [ p.standard-library ])
 
 or can be called as in the [Compiling Agda](#compiling-agda) section.
 
-If you want to use a library in your home directory (for instance if it is a development version) then typecheck it manually (using `agda.withPackages` if necessary) and then override the `src` attribute of the package to point to your local repository.
+If you want to use a different version of a library (for instance a development version)
+override the `src` attribute of the package to point to your local repository
+
+```
+agda.withPackages (p: [
+  (p.standard-library.overrideAttrs (oldAttrs: {
+    version = "local version";
+    src = /path/to/local/repo/agda-stdlib;
+  }))
+])
+```
+
+You can also reference a GitHub repository
+```
+agda.withPackages (p: [
+  (p.standard-library.overrideAttrs (oldAttrs: {
+    src =  fetchFromGitHub {
+      repo = "agda-stdlib";
+      owner = "agda";
+      rev = "v${version}";
```

Comment:
`version` isn't set here

Reply:
If the hash is a concrete version, set `version` to something concrete as well, I'd recommend

Reply:
thx